requirements.txt

```
@@ -1,3 +1,3 @@
-numpy >= 1.7
+numpy >= 2.0.0; python_version > '3.8'
```

Thinking more about this, it appears that we use the `requirements.txt` file to specify the install dependencies, which means we shouldn't constrain to numpy 2.0 here, since users can still use numpy 1.x on newer Python, and we want to be compatible with that.

So I think we don't actually want to change this here - saying `numpy >= 1.7` should let numpy 2.0 be installed on newer Python versions while letting older Python versions still work.

---

```suggestion
numpy >= 1.7
```
Yes, it does appear we want to leave the install requires as 1.7+ and only modify the build requires.